recipes/libx11/all/conanfile.py

```diff
@@ -41,7 +40,9 @@ def configure(self):
         del self.settings.compiler.cppstd
 
     def package(self):
-        self.copy("COPYING.txt", dst="licenses")
+        tools.download("https://gitlab.freedesktop.org/xorg/lib/libx11/raw/master/COPYING", filename="COPYING")
```

Comment:
much better! :clap:

Reply:
Dont forget to remove the license file `git rm COPYING`

Reply:
thanks!